<launch>
    <param name="max_iter" value="30"/>
    <param name="step_size" value="0.1"/>
    <param name="ndt_res" value="1.0"/>
    <param name="trans_eps" value="0.01"/>
    <param name="cloud_set_leaf_size_" value="1.3"/>
    <param name="local_set_leaf_size_" value="0.6"/>
    <param name="dis_set_leaf_size_" value="0.5"/>

    <param name="match_method_" value="NDT"/>
    <rosparam command="load" file="$(find lidar_localization)/config/config.yaml" />

    <node pkg="rviz"  type="rviz"  name="rviz"  args="-d $(find lidar_localization)/rviz/front_end.rviz"></node>
    <node pkg="lidar_localization"  type="front_end_node"  name="front_end_node"  output="screen"></node>
</launch>